README.md

```diff
@@ -84,8 +81,11 @@
       userAccountFeatureFlagsUserId: string;
       erpType: string;
       erpUserId: string;
+      viewState: any;

```

Comment:
Why we need include viewState here ?

Reply:
This is used to propagate ViewState on initialisation and avoid a second call other that the required REQUIRE_CONTEXT. If an outlet is in a tabs for example and open later than init, it will open with the proper state 👍.

Reply:
Oh wait ... true it should not be exposed 😅, removed, thanks for noticing 😊